hibernate-core/src/main/java/org/hibernate/sql/ast/spi/AbstractSqlAstTranslator.java

```diff
@@ -2957,6 +2957,11 @@ public void visitTuple(SqlTuple tuple) {
 	}
 
 	protected final void renderCommaSeparated(Iterable<? extends SqlAstNode> expressions) {
+		if ( ! expressions.iterator().hasNext() ) {
+			renderEmptyExpressions();
+			return;
+		}
+
```

Comment:
What's the reason for this? I don't think it's necessary.

```suggestion
```

Reply:
The code change is not necessary as per your approach